### PR TITLE
Test on Node.js 6

### PR DESCRIPTION
**Pull request by devinivy**
Hoping to use this in a project that is tested on node v6, so just making sure all my deps are covered :)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 language: node_js
 node_js:
+  - '6'
   - '5'
```
Comment (kevva) [inline .travis.yml:4]: Could you remove this line?

**Comment by SamVerschueren** [comment]
@sindresorhus Not sure what your opinion was on removing testing on `5`? But I guess it could be safely removed.

**Comment by sindresorhus** [comment]
@SamVerschueren Yes, Node.js 5 is gone in a few months anyways.

**Comment by devinivy** [comment]
That was quick– cheers!

**Pull request closed without merging**